glossary.yaml

```diff
@@ -141,6 +141,8 @@ terms:
     name: Keosd
   - definition: "In the EOSIO based blockchain context Ricardian Contract is a digital document that accompanies a smart contract and defines the terms and conditions of an interaction between the smart contract and its users, written in human readable text, which is then cryptographically signed and verified. It is easily readable for both humans and programs, and aids in providing clarity to any situations that may arise in the interactions between smart contract and its users."
     name: "Ricardian Contract"
+  - definition: "An action within a validated transaction, that is, an action whose transaction was executed and pushed to a block, but may or may not be final yet."
```

Comment:
The definition itself does not sound to convey anything about `retried`. Mind to elaborate why it is called retired action?

Reply:
@bobgt